charts/posit-chronicle/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+name: posit-chronicle
+description: Helm chart for the Chronicle Server
+version: 0.1.0
+appVersion: 2023.11.0
+icon: https://rstudio.com/wp-content/uploads/2018/10/RStudio-Logo-Flat.png
+home: https://www.posit.co
+sources:
+  - https://github.com/rstudio/helm
+maintainers:
+  - name: sol-eng
+    email: docker@rstudio.com
+    url: https://github.com/rstudio/helm
+annotations:
+  artifacthub.io/license: MIT
+  artifacthub.io/links: |
+    - name: About Posit Team
+      url: https://posit.co/products/enterprise/team/
```

Comment:
This was a discussion on our team, we weren't sure about linking posit community if there wasn't any guarantee there'd be chronicle related content there yet. I think it was @atheriel and I discussing this

But I think it could be nice adding them, maybe we can get some chronicle community engagement going in `r-admin`. 

The Posit Teams adds is also great, chronicle probably will run best it's current form as a 1:1 per posit team setup.